lib/model_2_factory-rails.rb

```diff
@@ -1,2 +1,3 @@
+require 'model_2_factory/constants'
```

Comment:
__[rubocop]__
* The name of this source file (`model_2_factory-rails.rb`) should use snake_case. [:ref](https://github.com/bbatsov/ruby-style-guide#snake-case-files)

<!-- Comment by [SideCI](https://sideci.com) -->